tensorflow/tensorflow-engine/src/main/java/ai/djl/tensorflow/engine/TfNDManager.java

```diff
@@ -248,8 +255,22 @@ public TfNDManager newSubManager(Device device) {
         return manager;
     }
 
-    public TfOpExecutor opExecutor(String operation) {
-        return new TfOpExecutor(this, ((TfEngine) getEngine()).getEagerSession(), operation);
+    /** {@inheritDoc} */
+    @Override
+    public void close() {
+        super.close();
+        TFE_Context eagerSession = eagerSessionHandle.getAndSet(null);
+        if (eagerSession != null && !eagerSession.isNull()) {
+            eagerSession.close();
+        }
+    }
+
+    TfOpExecutor opExecutor(String operation) {
```

Comment:
maybe make the function public? The user can use it to call TF operator not in DJL